README.md

```diff
@@ -10,10 +10,39 @@ To see what's incoming, check the [open pull requests](https://github.com/greene
 
 This manuscript was build with [Manubot Rootstock](https://git.io/vQSvo), which provides a system for collaboratively writing scholarly manuscripts via GitHub.
 
-## Source
+Instructions for using Manubot Rootstock for your own manuscript are still evolving.
```

Comment:
We could cut these two lines about cloning Manubot Rootstock and refer potential users back to the manubot-rootstock repository for the latest instructions.